root_main.tf

```diff
@@ -25,7 +25,7 @@ module "ses-eu-west-1" {
   project               = var.project
   environment_full_name = lookup(local.environment_full_name_map, local.environment)
   hosted_zone_id        = module.route_53_zone.hosted_zone_id
-  dns_delegated         = local.environment == "prod" ? false : true
+  dns_delegated         = local.environment == "xxxx" ? false : true #replace xxxx with workspace for new environment
```

Comment:
Should this be "xxxx"?

Reply:
Thanks, I've commented out the line and made it clearer that this is an example for any new environments, for which DNS has not yet been delegated